index.js

```diff
@@ -1,19 +1,17 @@
 const express = require('express');
 const bodyParser = require('body-parser');
-const helmServer = require('./autom8s/helm-server');
-const portServer = require('./autom8s/port-server');
-const ingressServer = require('./autom8s/ingress-server');
-const sampleApp = require('./sample/index');
+const helmController = require('./autom8s/helm-controller');
+const portController = require('./autom8s/port-controller');
+const ingressController = require('./autom8s/ingress-controller');
 
 const app = express();
 
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
-app.use('/', helmServer);
-app.use('/', portServer);
-app.use('/', ingressServer);
-app.use('/', sampleApp);
+app.use('/', helmController);
+app.use('/', portController);
+app.use('/', ingressController);
```

Comment:
I have told Itye that a better practice would be to have all the HTTP commands here (app.post, app.get, etc.) which makes very clear what is the API. And then you don't have xxx-service, but rather just xxx classes..
app.use hides the interface. 
I don't like it.